### DISCUSSION TITLE
Starting the Unity program for the second time, an error "EntryPointNotFoundException" is reported

### DISCUSSION THREAD

**Discussion by erwincoumans** [bulletphysics/bullet3#4083] [General]
### Discussed in https://github.com/bulletphysics/bullet3/discussions/4081

<div type='discussions-op-text'>

<sup>Originally posted by **LuckMonkey2000** December  6, 2021</sup>
I use Unity to call the dll exported by Bullet. Debugging requires frequent replacement of the dll, so when the Unity program ends, I will use FreeLibrary() to release the dll. But sometimes, when the Unity program is started for the second time, an error "EntryPointNotFoundException:" will be reported. How to solve this problem?
![image](https://user-images.githubusercontent.com/25970563/144963283-712dc8b5-3a8e-4944-a105-c72ca9941651.png)
</div>

**Comment by YoruCathy** [comment]
Hi, by any chance is there a solution to this?
